domains/transfer/Transfer.API/API/Controllers/ConnectionsController.cs

```diff
@@ -17,10 +18,47 @@ namespace API.Controllers;
 public class ConnectionsController : Controller
 {
     private readonly IConnectionRepository connectionRepository;
+    private readonly IConnectionInvitationRepository connectionInvitationRepository;
 
-    public ConnectionsController(IConnectionRepository connectionRepository)
+
+    public ConnectionsController(IConnectionRepository connectionRepository, IConnectionInvitationRepository connectionInvitationRepository)
     {
         this.connectionRepository = connectionRepository;
+        this.connectionInvitationRepository = connectionInvitationRepository;
+    }
+
+    [HttpPost]
+    [ProducesResponseType(typeof(Connection), 201)]
```

Comment:
404 Not found is missing

Reply:
Fixed: https://github.com/Energinet-DataHub/energy-origin/pull/439/commits/0001b7c97ce1089d6dd6a44a31e8d0bb85a2f279